frontend/src/pages/Contributions.jsx

```
@@ -0,0 +1,151 @@
+      <div className="contributions-content">
+        <h1 style={{color: '#6c2bb3'}}>All the exam preparations you need in one place</h1>
+        <p className="subtitle">From critical skills to exam topics, edusphere supports your development.</p>
+        
+        <div className="sub-navbar">
+          <div className="tags-container">
+            {tags.map((tag) => (
+              <button 
+                key={tag}
+                className={`nav-tag ${activeTag === tag ? 'active' : ''}`}
+                onClick={() => setActiveTag(tag)}
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)